partner.css

```
@@ -446,7 +446,7 @@ and (-webkit-device-pixel-ratio : 3) {
     img#tooltip-phone.tooltip {
         background: url(question.svg) no-repeat right center !important;
         object-position: -99999px 99999px;
-        left: 82px;
+        left: 93px;
     }
 
     img#tooltip-registrant_name.tooltip{
```

ugh I missed this when cleaning up - please add a space here
```suggestion
    img#tooltip-registrant_name.tooltip {
```